Backend/app/routes/brand_dashboard.py

```
@@ -0,0 +1,1132 @@
+    brand_id: str = Query(..., min_length=36, max_length=36, regex=r"^[a-fA-F0-9\-]{36}$", description="Brand user ID (UUID)")
+):
+    """
+    Delete a campaign
+    """
+    try:
+        # Verify campaign belongs to brand
+        existing = supabase.table("sponsorships").select("*").eq("id", campaign_id).eq("brand_id", brand_id).execute()
+        if not existing.data:
+            raise HTTPException(status_code=404, detail="Campaign not found")
+        
+        response = supabase.table("sponsorships").delete().eq("id", campaign_id).execute()
```

Ruff (0.13.1)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)